vendor/vendor.json

```diff
@@ -106,6 +106,12 @@
 			"version": "v12.5.0-beta",
 			"versionExact": "v12.5.0-beta"
 		},
+		{
+			"checksumSHA1": "OXG8AijhiNimr1zNSJ85SnfcrI8=",
+			"path": "github.com/Azure/azure-sdk-for-go/services/hdinsight/mgmt/2015-03-01-preview/hdinsight",
+			"revision": "7b986a3a0511159a0a7877f0d7d775bdfe916d29",
+			"revisionTime": "2017-12-16T00:10:16Z"
```

Comment:
can we pin to using `v12.5.0-beta` of the Azure SDK for the moment? we can do that via:

```
govendor update github.com/Azure/azure-sdk-for-go/services/hdinsight/mgmt/2015-03-01-preview/hdinsight.@=v12.5.0-beta
```